starter-code/scripts/controllers/articleController.js

```diff
@@ -4,8 +4,9 @@
   var articleController = {};
 
   articleController.reveal = function() {
-    /* TODO: Use your DOM skills to reveal only the articles section! */
+    /* TODO: DONE Use your DOM skills to reveal only the articles section! */
+    $('#about').hide();
+    $('#articles').fadeIn(1000);
```

Comment:
Perfect! To make this more extensible (if you were to add other sections), you could also hide all of the sections with a class of .tab-content and then only reveal the section associated with the controller.